ui/app/components/pending-tx.js

```diff
@@ -74,18 +72,24 @@ PendingTx.prototype.getTotal = function () {
   const { params = [] } = decodedData || {}
   const { name, value } = params[1] || {}
   const amountBn = name === '_value'
-    ? new BN(value)
-    : hexToBn(txParams.value)
+    ? value
+    : txParams.value
```

Comment:
Seems to be unrelated to linting

Reply:
Yes, unrelated to linting. This branch had commits on it that it should not have... due to the reverting of some branches that was done last week. I have removed those commits.